io.github.paledega.alpine-rootfs.yaml

```
@@ -0,0 +1,30 @@
+app-id: io.github.paledega.alpine-rootfs
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: alpine
+finish-args:
+  - --talk-name=org.freedesktop.Flatpak
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=home
+modules:
+  - name: build
```

```suggestion
  - name: alpine
```

at least make is relatable.

---

thx